views/account/profile/main.js

```diff
@@ -0,0 +1,56 @@
+// JQuery Datatables Initialization
+$(document).ready(function () {
+  var table = $("#casual-table").DataTable();
+  new $.fn.DataTable.Responsive(table);
+});
+$(document).ready(function () {
+  var table = $("#professional-table").DataTable();
+  new $.fn.DataTable.Responsive(table);
+});
+
+let casual_table = document.getElementById("casual-table-body");
+let professional_table = document.getElementById("professional-table-body");
+
+// These are dummy values for now; we would need to pull the casual and professional info from the db
+let casual_data_labels = ["Instagram", "Facebook"];
```

Comment:
Sounds good. Once the endpoint is set up for getting all of a user's accounts, this is where we will add that API call.